ksp_plugin/plugin.cpp

```diff
@@ -937,18 +941,75 @@ void Plugin::ComputeAndRenderNodes(
                Vector<double, Navigation>({0, 0, 1}),
                ascending_trajectory,
                descending_trajectory);
-  ascending = renderer_->RenderPlottingTrajectoryInWorld(
```

Comment:
Please keep this formatted the way it was.  It will make the diff easier to review.